CIP-0027/README.md

```diff
@@ -53,6 +53,23 @@ A new tag of 777 is proposed for this implementation.  The community guidelines
 	}
 }
 
+## Example JSON with multiple addresses
+
+{
+	"777": [
+	{
+		"rate": "0.2",
+		"addr": "addr1v9nevxg9wunfck0gt7hpxuy0elnqygglme3u6l3nn5q5gnq5dc9un"
+	},
+	{
+		"rate": "0.12",
+		"addr": [
+			"addr1q8g3dv6ptkgsafh7k5muggrvfde2szzmc2mqkcxpxn7c63l9znc9e3xa82h",
+			"pf39scc37tcu9ggy0l89gy2f9r2lf7husfvu8wh"
+		]
+	}]
+}
```

Comment:
Make sure to put that example in a proper blockquote (using \`\`\`) so that it renders properly. Example:

```json
{
	"777": [
	{
		"rate": "0.2",
		"addr": "addr1v9nevxg9wunfck0gt7hpxuy0elnqygglme3u6l3nn5q5gnq5dc9un"
	},
	{
		"rate": "0.12",
		"addr": [
			"addr1q8g3dv6ptkgsafh7k5muggrvfde2szzmc2mqkcxpxn7c63l9znc9e3xa82h",
			"pf39scc37tcu9ggy0l89gy2f9r2lf7husfvu8wh"
		]
	}]
}
```